src/parser.ts

```diff
@@ -696,6 +760,10 @@ function validateArity(name: string, arity: GroqFunctionArity, count: number) {
   }
 }
 
+function argumentShouldBeSelector(namespace: string, args: NodeTypes.ExprNode[]) {
+  return namespace == 'diff' && AbortSignal.length == 2
```

Comment:
`AbortSignal` should be `args`, right?

Reply:
Yes absolutely, don't know how that got in there.